modules/ui/src/main/scala/lucuma/ui/aladin/package.scala

```diff
@@ -60,13 +61,19 @@ object CooFrame:
       case "j2000d"   => Some(J2000d)
       case "galactic" => Some(Galactic)
 
+enum ImageSurvey(val tag: String, val name: String, val id: String) derives Enumerated:
```

Comment:
Do you think it's worth to move into `ImageSurvey.scala` and have a companion object with a `def defaultFor(i: Instrument)`? (which is now in the state initializer in `AladinContainer`)

Reply:
We have the business logic there anyway, only that it's buried in a state initializer instead of being more explicit.

I think that organizing it is a step forward, short of moving the whole thing to core.